AsyncDisplayKit/ASEditableTextNode.mm

```diff
@@ -16,16 +16,31 @@
 #import "ASTextNodeWordKerner.h"
 #import "ASThread.h"
 
-//! @abstract This subclass exists solely to ensure the text view's panGestureRecognizer never begins, because it's sporadically enabled by UITextView. It will be removed pending rdar://14729288.
-@interface _ASDisabledPanUITextView : UITextView
+//! @abstract This subclass forces the parent UITextView's scrollEnabled property to always be true. Instead, it disables the panGestureRecognizer when scrollEnabled is set to false. This ensures that the contentSize is caculated correctly.
+//! See issue: https://github.com/facebook/AsyncDisplayKit/issues/1063
```

Comment:
Break out the abstract into a few lines.  Pretty good as-written!  I would add a clarification / maybe replace some of your words with:

"As originally reported in rdar://14729288, when scrollEnabled = NO, UITextView does not calculate its contentSize.  This makes it difficult for a client to embed a UITextView inside a different scroll view with other content (setting scrollEnabled = NO on the UITextView itself, because the containing scroll view will handle the gesture)...because accessing contentSize is typically necessary to perform layout.  Apple later closed the issue as expected behavior.  This works around the issue by ensuring that contentSize is always calculated, while still providing control over the UITextView's scrolling."